neurom/features/bifurcationfunc.py

```diff
@@ -125,3 +134,58 @@ def partition_pair(bif_point):
     n = float(sum(1 for _ in bif_point.children[0].ipreorder()))
     m = float(sum(1 for _ in bif_point.children[1].ipreorder()))
     return (n, m)
+
+
+def partition_asymmetry_length(bif_point):
+    '''Calculate a variant of the partition asymmetry based on path lengths
+
+    This asymmetry is defined as the absolute difference between total lengths of
+    the tress after each daughter.
+
+    WARNING: as is, it will not be of interest, as it has to be rescaled by the
+    total lenght of the neurite'''
+    _raise_if_not_bifurcation(bif_point)
+
+    n = float(sum(sec.length for sec in bif_point.children[0].ipreorder()))
+    m = float(sum(sec.length for sec in bif_point.children[1].ipreorder()))
+    return abs(n - m)
+
+
+def sibling_ratio(bif_point, method='first'):
+    '''Calculate the sibling ratio of a bifurcation point
+
+    The sibling ratio is the ratio between the diameters of the
+    smallest and the largest daughter. It is a real number between
+    0 and 1. Method argument allows one to consider mean diameters
+    along the daughter section instead of diameter of the first point. '''
+    _raise_if_not_bifurcation(bif_point)
+
+    if method == 'first':
+        n = bif_point.children[0].points[0, COLS.R]
+        m = bif_point.children[1].points[0, COLS.R]
+    elif method == 'mean':
+        n = _get_mean_radius(bif_point.children[0])
+        m = _get_mean_radius(bif_point.children[1])
+    else:
+        raise NeuroMError('Please provide a valid method for sibling ratio, found{}'.format(
+            method))
```

Comment:
```suggestion
         raise ValueError('Please provide a valid method for sibling ratio, found %s', method)
```
Using %s instead of .format is best practice for exceptions